examples/remote-import-import.phpi

```diff
@@ -0,0 +1,12 @@
+<?php
+
+use function Castor\import;
+use function Castor\wait_for_import;
+
+import('composer://doctrine/collections');
+
+wait_for_import();
+
+if (class_exists('Doctrine\Common\Collections\ArrayCollection')) {
```

Comment:
needed because we have no remote option in some places where the vendor are not loaded yet, but it should not be need in real app